services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/SingleRateSummarySection.tsx

```diff
@@ -225,7 +230,8 @@ export const SingleRateSummarySection = ({
             )
         }
     }
-
+    const parentContractSubmissionID =
```

Comment:
@macrael Does this seem like a reasonable path to get the contract submission ID of the contract that originally was created with that rate? Or is there some other way to get to the parent contract

Reply:
Shouldn't it be `rate.revisions[rate.revisions.length -1]...`. We should probably include it in the rate itself from the API rather than calculate it everywhere

Reply:
Ah yes, that sounds more right. It's the first added to the array 😮‍💨  I will make that change. TY